birthdays-project/BirthdayApp/admin.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib import admin
+from django.db import models
+from django.forms import TextInput, Textarea
+
+from .models import User, Birthdate
+
+
+# Modifies the appearance of the textfields in the admin section.
+class ModAdmin(admin.ModelAdmin):
```

Comment:
This is the admin for the `User` model, so the name should be `UserAdmin` (or `PersonAdmin` if you follow my suggestion to refactor the model name)